plugin-interface/src/main/scala/mesosphere/marathon/plugin/scheduler/SchedulerPlugin.scala

```diff
@@ -9,7 +9,7 @@ import org.apache.mesos.Protos.Offer
   * Allows to use external logic to decline offers.
   */
 trait SchedulerPlugin extends Plugin {
- 
+
```

Comment:
rm new line

Reply:
This was a remove of an unwanted whitespace to prevent a reformat in the next time.